regex.c

```diff
@@ -5484,12 +5482,12 @@ re_comp (s)
 
   if (!re_comp_buf.buffer)
     {
-      re_comp_buf.buffer = (unsigned char *) malloc (200);	/* __MEM_CHECKED__ */
+      re_comp_buf.buffer = malloc (200);	/* __MEM_CHECKED__ */
```

Comment:
Shouldn't these calls also be using the `safe_*` functions? Or is that something for a different patch?